src/routes/index.ts

```diff
@@ -216,6 +216,14 @@ router.route(config.BENEFICIAL_OWNER_OTHER_URL)
   .get(beneficialOwnerOther.get)
   .post(...validator.beneficialOwnerOther, checkValidations, beneficialOwnerOther.post);
 
+router.route(config.BENEFICIAL_OWNER_OTHER_WITH_PARAMS_URL)
```

Comment:
we should also update the 
config.BENEFICIAL_OWNER_OTHER_URL + config.REMOVE + config.ID
route as well to make sure the remove button works as well as the change button

Reply:
hang on, i think it is already there

Reply:
yes can see it, ignore this .

Reply:
Don't know how to get this to work, when remove is clicked the ids are undefined not sure what is wiping them out.